pkg/acs/calacs/acsccd/doccd.c

```diff
@@ -438,6 +439,10 @@ int DoCCD (ACSInfo *acs_info) {
         if (acs_info->printtime)
             TimeStamp("SINKCORR complete", acs_info->rootname);
     }
+
+    if (!OmitStep(acs[0].sinkcorr))
+        if (sinkHistory(&acs[0], x[0].globalhdr))
```

Comment:
This should really be ``if ((status = sinkHistory(&acs[0], x[0].globalhdr)))``. I'm not convinced that ``status`` is as global as it needs to be and either way, we shouldn't be using a global for this.

Same goes for all the others in this PR.